package.json

```diff
@@ -53,13 +53,15 @@
 		"rc-slider": "^8.4.0",
 		"react": "16.8.1",
 		"react-color": "^2.18.0",
+		"react-cropper": "^2.1.8",
 		"react-custom-scrollbars": "^4.2.1",
 		"react-dom": "16.8.1",
 		"react-dropzone": "^4.2.8",
 		"react-linkify": "^0.2.2",
 		"react-popper": "^2.2.3",
 		"react-redux": "^5.0.6",
 		"react-select": "^3.0.0",
+		"react-sweetalert2": "^0.5.1",
```

Comment:
We do already pull in the generic version of `sweetalert2` below - is there a big reason to use the react specific version? (I didn't know it existed to be honest) - I'd prefer we have just one of them, so either by using the generic version here or swapping all the rest of the instances to use the react version.

Reply:
`react-sweetalert2` exposes the react component `<SweetAlert2/>` which allowed me to have another react component (`<Cropper/>`) in the body of the alert. If there is another way of achieving this without the react version it can be switched.